frontend/src/components/Form.js

```diff
@@ -86,6 +88,24 @@ const Form = ({ setNumberOfErrors, setErrorsExistCounter }) => {
     });
   };
 
+  const check_dl_status = (dl) => {                 
```

Comment:
I think this function would somehow need to hook into the form's validation methods kind of like this sandbox: https://codesandbox.io/s/react-hook-form-async-field-validation-w1chw

Reply:
@naomiaro Can you please elaborate. I am not able to check status using validation

Reply:
I'm not fully sure of this lib, if it's too difficult/can't be done let's just leave this part as is for now!